real_estate/models/estate_property.py

```diff
@@ -0,0 +1,129 @@
+from odoo import api, exceptions, fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Property"
+
+    # Description
+    name = fields.Char(
+        string="Property Name",
+        required=True)
+    description = fields.Text(
+        string="Property Description")
+    postcode = fields.Char(
+        string="Postcode") 
+    date_availability = fields.Date(
+        string="Availabile From",
+        copy=False,
+        default=lambda self:fields.Datetime.today() + relativedelta(months=3)
+    )
+
+    expected_price = fields.Float(
+        string="Expected Price",
+        required=True)
+    selling_price = fields.Float(
+        string="Selling Price",
+        readonly=True, 
+        copy=False)
+    bedrooms = fields.Integer(
+        string="Bedrooms",
+        default=2)
+    living_area = fields.Integer(
+        string="Living Area")
+    facades = fields.Integer(
+        string="Facades")
+    garage = fields.Boolean(
+        string="Garage")
+    garden = fields.Boolean(
+        string="Garden")
+    garden_area = fields.Integer(
+        string="Garden Area")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ('north', "North"),
+            ('south', "South"),
+            ('east', "East"), 
+            ('west', "West")
+        ]
+    )
+    total_area = fields.Float(
+        compute='_compute_total_area')
+    active = fields.Boolean(
+        string="Active", 
+        default=True)
+    state = fields.Selection(
+        string='State',
+        selection=[
+            ('new', "New"),
+            ('offer_received', "Offer Received"),
+            ('offer_accepted', "Offer Accepted"),
+            ('sold', "Sold"),
+            ('cancelled', "Cancelled")
+        ],
+        required=True,
+        default="new",
+        copy=False
+    )
+    
+    property_type_id = fields.Many2one(
+        'estate.property.type',
+        string="Property Type")
+    property_tag_ids = fields.Many2many(
+        'estate.property.tag',
+        string="Property Tags")
+    offer_ids = fields.One2many(
+        'estate.property.offer',
+        'property_id')
+    user_id = fields.Many2one(
+        'res.users', 
+        string='Salesperson',
+        index=True,
+        tracking=True,
+        default=lambda self: self.env.user)
```

Comment:
```suggestion
        default=lambda self: self.env.user
    )
```
Make sure adding the ending parenthesis in a newline makes it more readable and also makes it easy to add any new attribute in the existing code